.github/workflows/test.yml

```diff
@@ -40,6 +40,8 @@ jobs:
     - run: which colcon
     - run: which rosdep
     - run: which vcs
+    - if: runner.os != 'Linux'
```

Comment:
I think we can drop this, we should have it in the path for all platforms.

If this fails for any reason, I'd be OK adding this with a FIXME, but hopefully other platforms aren't broken ;)

Reply:
I added a `FIXME`. Currently the cppcheck is not in the system path at this step for Linux. However, it won't render as a problem when running `colcon test`. It seems the `cppcheck` gets installed later when `action-ros-ci` runs.